Resources/Textures/_NF/Objects/Decoration/Flora/flora_tree_palm.rsi/meta.json

```
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
```

```suggestion
    "license": "CC-BY-NC-SA-3.0",
```
I goofed, the goon assets are specifically non-commercial